### DISCUSSION TITLE
Add support for JMAP in addition to CalDAV

### DISCUSSION THREAD

**Discussion by dumblob** [bitfireAT/icsx5#58] [Ideas]
>JMAP is a quickly spreading standard for clients to connect to calendar, address, mail, etc. stores. It therefore intends to primarily accompany or replace CalDAV, CardDAV, IMAP, and SMTP. It is also designed to be more generic such that it can be extended in the future (this allows replacing CardDAV/CalDAV etc.). Note JMAP does not replace MTA-to-MTA SMTP transmission.

https://jmap.io/spec-calendars.html

About the JMAP working group at the IETF: https://datatracker.ietf.org/wg/jmap/about/

Thank you!

**Comment by dumblob** [comment]
@marclaporte might be of interest for you :wink:.

**Reply by roke-julian-lockhart** [reply]
[`ietf-tools/datatracker/issues/6979#issuecomment-4415932647`](https://github.com/ietf-tools/datatracker/issues/6979#issuecomment-4415932647) may be of interest to you both.
